frontend/src/component/releases/ReleaseManagement/ReleasePlanTemplateCardMenu.tsx

```diff
@@ -16,7 +16,8 @@ import { TemplateDeleteDialog } from './TemplateDeleteDialog';
 
 export const ReleasePlanTemplateCardMenu = ({
     template,
-}: { template: IReleasePlanTemplate }) => {
+    clickHandler,
```

Comment:
How about something like `onClick` instead? That's a bit more consistent with how we identify event handlers.